### DISCUSSION TITLE
Why am I no longer able to update Italian translations with Pull Request?

### DISCUSSION THREAD

**Discussion by morphinestyle** [akeeba/panopticon#659] [General]
Why am I no longer able to update translations with Pull Request?
GitHub return: "Pull request creation failed. Validation failed: must be a collaborator".

**Comment by nikosdion** [comment]
That's weird. I haven't made any changes, and I definitely got a number of translation PRs last week from someone who is _not_ a collaborator.

Can you try creating a new fork of the repository? I am wondering if something is broken in your fork.

**Comment by morphinestyle** [comment]
I clicked again on FORK, and tried to create a PULL REQUEST, but nothing changes.
Unfortunately, I'm not very good at using GitHub, but now it seems that something has changed.

**Comment by morphinestyle** [comment]
In the latest translation, I have made numerous refinements and adjustments, in addition to translating all the new sentences.
How can we proceed? How can I send you the file?

**Comment by NetSecond** [comment]
You have to made the translation over your fork and when finished, on save you create automatically a commit.
This commit(s) you can "pull request".

I think, it is not possible to create a new empty pull request.
Sorry, I am not so good in GitHub things and ignore it, when I am wrong ....

**Reply by nikosdion** [reply]
This sounds about right. That's the easiest way to create a pull request from GitHub's interface.

@morphinestyle Is this what you're trying to do? If not, how did you try doing the PR?

**Reply by morphinestyle** [reply]
It seems to have worked now.
But I haven't done it this way the past times: it seems that some mechanism, for some reason, has changed compared to the Pull Requests of a few months ago.

In any case we should be there.

**Reply by morphinestyle** [reply]
Can you confirm that everything is correct?

**Reply by nikosdion** [reply]
I got the pull request. There are some merge conflicts as you seem to have based it off an earlier point in time. I will resolve them, don't worry.
